zokrates_cli/tests/code/precedence.expected.witness

```diff
@@ -0,0 +1 @@
+~out_0 1
```

Comment:
There should be a `precedence.arguments.json` file with the arguments passed to the function, here `[]` I guess.

Reply:
I see, wasn't able to test the running testing integration as it wasn't working. Regarding the expected.witness: Left the file empty since the cli returns nothing for the witness when running this